sdk/java/src/main/java/com/chain/api/Account.java

```diff
@@ -253,4 +253,94 @@ public Builder setTags(Map<String, Object> tags) {
       return this;
     }
   }
+
+  /**
+   * Use this class to create a {@link Receiver} under an account.
+   */
+  public static class ReceiverBuilder {
+    @SerializedName("account_alias")
+    public String accountAlias;
+
+    @SerializedName("account_id")
+    public String accountId;
+
+    @SerializedName("expires_at")
+    public Date expiresAt;
+
+    /**
+     * Specifies the account under which the receiver is created. You must use
+     * this method or @{link ReceiverBuilder#setAccountId}, but not both.
+     *
+     * @param alias The unique alias of the account.
+     * @return The updated ReceiverBuilder object.
```

Comment:
Might be worth looking at this: http://www.oracle.com/technetwork/articles/java/index-137868.html

Couple things that I like from there:
* Use "this" instead of "the" when referring to an object created from the current class.
* re `@param`: . The description begins with a lowercase letter if it is a phrase (contains no verb), or an uppercase letter if it is a sentence. End the phrase with a period only if another phrase or sentence follows it.

Even the stdlib's javadocs aren't particularly opinionated. I do like the lowercase, phrases for simple `@param` tags like here: [java.util.concurrent.atomic.AtomicInteger](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/atomic/AtomicInteger.html) But, maybe full sentences shouldn't be prohibited for more complicated parameters.